pushapkscript/requirements/base.txt

```diff
@@ -196,19 +199,19 @@ github3.py==2.0.0 \
     --hash=sha256:7e153e5dc559f27e4d15c91571a2d6e1522c9faa4406e188459d9731f0b98875 \
     --hash=sha256:8dd4ac612fd60cb277eaf6e2ce02f68dda54aba06870ca6fa2b28369bf39aa14
     # via scriptworker
-google-api-core==1.31.0 \
-    --hash=sha256:7c8ba88e2b893ef4f67d67e229aade51a2db5053023b73b1394a5ee3dcdb561c \
-    --hash=sha256:a9f979b5c6a9cad31a4120ca6be712df76adc32336a7bf4bc2f4331a1b527fe7
+google-api-core==2.0.0 \
+    --hash=sha256:428805de17b48ca1af2fdb5bbfc2334e1bbcb0ea4a16506fa1337fb29c8f37c8 \
+    --hash=sha256:bd9eb0709f4e10dd6fddb32fd0788a190b434426c258be6e00ef40da136d768d
     # via google-api-python-client
-google-api-python-client==2.13.0 \
-    --hash=sha256:345bb7249ac9af93a6c75e4adb93a41139459945aee61e611a5c8e5b146cbef7 \
-    --hash=sha256:b3874333cabc5ce46dcda2da48f1b258e227a9ae61688211e277e11270386307
+google-api-python-client==2.18.0 \
+    --hash=sha256:6e990fc4d0419c2011f75ca5c2762efbb7eb4def67bbe2f1b98a8ccb73117bf5 \
+    --hash=sha256:a25661ec6cf4c159f41fe9c061c2bee31b2dddaf2ad787e23617048a25b53842
     # via
     #   -r requirements/base.in
     #   mozapkpublisher
-google-auth==1.33.0 \
-    --hash=sha256:9bd25d835c01ca3dd9045ea0b50f036e3fe3868ee2c5a9773a661925bbdf46ca \
-    --hash=sha256:a756a33978fac611e4f00b69715b80e35467c30cc6262132d29d33a0e398ac55
+google-auth==2.0.1 \
```

Comment:
and here https://github.com/googleapis/google-auth-library-python/blob/master/CHANGELOG.md#200-2021-08-16